From 3a525c406d695605e94063328d3d2bda063ba27e Mon Sep 17 00:00:00 2001
From: zrj <rimvydas.jasinskas@gmail.com>
Date: Sat, 3 Aug 2019 21:30:51 +0300
Subject: [PATCH 6/8] ppp(8): Disable RADIUS support.

 If there are still users making use of RADIUS support in ppp(8),
 then it could be added back directly into ppp(8).

 Add __unused for few arguments to pet WARNS.
---
 usr.sbin/ppp/Makefile | 12 ++++++------
 usr.sbin/ppp/mppe.c   |  6 +++---
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/usr.sbin/ppp/Makefile b/usr.sbin/ppp/Makefile
index a508731e15..2f321abc52 100644
--- a/usr.sbin/ppp/Makefile
+++ b/usr.sbin/ppp/Makefile
@@ -60,13 +60,13 @@ CFLAGS+=-DNODES
 SRCS+=		chap_ms.c mppe.c
 .endif
 
-.if defined(NORADIUS)
+#.if defined(NORADIUS)
 CFLAGS+=-DNORADIUS
-.else
-SRCS+=	radius.c
-LDADD+=	-lradius
-DPADD+= ${LIBRADIUS}
-.endif
+#.else
+#SRCS+=	radius.c
+#LDADD+=	-lradius
+#DPADD+= ${LIBRADIUS}
+#.endif
 
 .if defined(WANT_NETGRAPH7)
 CFLAGS+=-DWANT_NETGRAPH7
diff --git a/usr.sbin/ppp/mppe.c b/usr.sbin/ppp/mppe.c
index e948832092..7a1f0deb29 100644
--- a/usr.sbin/ppp/mppe.c
+++ b/usr.sbin/ppp/mppe.c
@@ -471,7 +471,7 @@ MPPERequired(struct fsm *fp)
 }
 
 static u_int32_t
-MPPE_ConfigVal(struct bundle *bundle, const struct ccp_config *cfg)
+MPPE_ConfigVal(struct bundle *bundle __unused, const struct ccp_config *cfg)
 {
   u_int32_t val;
 
@@ -677,7 +677,7 @@ MPPE_InitState(struct fsm_opt *o)
 }
 
 static void *
-MPPEInitInput(struct bundle *bundle, struct fsm_opt *o)
+MPPEInitInput(struct bundle *bundle __unused, struct fsm_opt *o)
 {
   struct mppe_state *mip;
 
@@ -733,7 +733,7 @@ MPPEInitInput(struct bundle *bundle, struct fsm_opt *o)
 }
 
 static void *
-MPPEInitOutput(struct bundle *bundle, struct fsm_opt *o)
+MPPEInitOutput(struct bundle *bundle __unused, struct fsm_opt *o)
 {
   struct mppe_state *mop;
 
-- 
2.22.0

